app/controllers/organisation/BusinessNotIdentifiedController.scala

```diff
@@ -32,20 +34,27 @@ class BusinessNotIdentifiedController @Inject() (
   standardActionSets: StandardActionSets,
   val controllerComponents: MessagesControllerComponents,
   appConfig: FrontendAppConfig,
-  view: BusinessNotIdentifiedView
+  view: BusinessNotIdentifiedView,
+  viewSoleTrader: SoleTraderNotIdentifiedView
 ) extends FrontendBaseController
-    with I18nSupport {
+    with I18nSupport
+    with Logging {
 
   def onPageLoad(): Action[AnyContent] = standardActionSets.identifiedUserWithData() {
     implicit request =>
-      val startUrl = controllers.routes.IndexController.onPageLoad.url
+      val startUrl     = controllers.routes.IndexController.onPageLoad.url
+      val reporterType = request.userAnswers.get(ReporterTypePage)
 
       val contactLink: String = request.userAnswers.get(ReporterTypePage) match {
```

Comment:
You can save a `get` here by just matching on the `reporterType` val from the line above.